src/scenes/Contacts/components/Form/Form.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+
+import styles from './Form.module.scss';
+
+const Form = () => {
+  let [data, setData] = useState({
+    name: '',
+    company: '',
+    email: '',
+    message: '',
+    files: [],
+  });
+
+  // const key = 'https://getform.io/f/f65d5c0b-3ff7-40f7-a418-3ea72cd680cd';
+
+  // const isNotEmpty = value => value !== '';
+
+  // const isCompanyEmailValid = email => {
+  //   const regexp = new RegExp(/[^@]+@[^.]+\..+/g);
+  //   return regexp.test(email);
+  // };
+
+  // const isFileSizeValid = (fileType, fileSize) => {
+  //   const fileSizeInMB = fileSize / 1024 / 1024;
+
+  //   return [ALLOWED_FILES.PDF.MIME_TYPE, ALLOWED_FILES.PPT.MIME_TYPE].includes(
+  //     fileType
+  //   )
+  //     ? fileSizeInMB < 10
+  //     : fileSizeInMB < 1;
+  // };
+
+  const handleSubmit = () => {
+    console.log(data);
+  };
+
+  const handleInputFileChange = data => {
+    console.log(data);
+  };
+
+  return (
+    <div className={styles.container}>
+      <form
+        className={styles.form}
+        action="#"
+        method="POST"
+        encType="multipart/form-data"
+        onSubmit={handleSubmit}
+      >
+        <h3 className={styles.formTitle}>REQUEST A QUOTE</h3>
+        <div className={styles.inputWrapper}>
+          <input
+            className={styles.input}
+            type="text"
+            name="name"
+            id="name"
+            require="true"
+            onChange={event => setData({ name: event.target.value })}
+          />
+          <label className={styles.placeholder} htmlFor="name">
+            Full Name
+          </label>
+        </div>
+        <div className={styles.inputWrapper}>
+          <input
+            className={styles.input}
+            type="text"
+            name="company"
+            id="company"
+            onChange={event => setData({ company: event.target.value })}
+          ></input>
+          <label className={styles.placeholder} htmlFor="company">
+            Company
+          </label>
+        </div>
+        <div className={styles.inputWrapper}>
+          <input
+            className={styles.input}
+            type="email"
+            name="email"
+            id="email"
+            require="true"
+            onChange={event => setData({ email: event.target.value })}
```

Comment:
1)This function should be transferred from the JSX to the component (since each time a component is drawn when a component is drawn)
2)initial state is:
{
    name: '',
    company: '',
    email: '',
    message: '',
    files: [],
  }, but when you make setData({ email: event.target.value }), you get { email: 'something value' }, you rewhite all other fields